.github/scripts/upload_smoke_test.sh

```diff
@@ -0,0 +1,2 @@
+bucket=$1
+aws s3api put-object --bucket $bucket --key "smoke.html" --body processed_smoke.html --content-type "text/html"
```

Comment:
question: Would it be difficult to version the smoke test file? E.g., `smoke-1.5.1.html`? This way we could (1) retain older versions for testing, and (2) it would be easy to verify which versions have been tested.

This isn't a blocker, but something to consider if it's easy to do.

Reply:
In order to do so, I think the changes need to happen on (1) CDK to edit the bucket with versioning characteristics (2) `upload_smoke_test.sh` and `cd.yaml`.

I think it's better to have the fix incorporated with this, since otherwise we will have to keep the enhancement in our backlog (which might get overlooked).

I will add the changes with the next revision.

Reply:
Clarification: Bucket versioning is not needed - we expect the bucket to hold different versions of the smoke test, with the naming convention of `*.cloudfront.net/smoke-{version}.html`
Verified testing in gamma